e2e/fixtures/unversioned/MODULE.bazel

```diff
@@ -1,4 +1 @@
-module(
-    name = "unversioned",
-    version = "0.0.0",
-)
\ No newline at end of file
+module(name = "unversioned")
```

Comment:
Could you add an additional test instead of replacing the existing one?

Reply:
Yeah this particular fixture is used to test version patching from 0.0.0 which is a common use case.

Reply:
added